ac_dc/oscar_sample_filter.py

```diff
@@ -403,13 +403,26 @@ def __init__(
         path_kenlm_model,
         num_proc,
         path_dir_save_oscar,
+        use_registry,
+        registry_data_files,
     ):
         self.lang_oscar_id = lang_oscar_id
         self.path_fasttext_model = path_fasttext_model
         self.path_kenlm_model = path_kenlm_model
-        self.ds = load_dataset(
-            "oscar", f"unshuffled_deduplicated_{self.lang_oscar_id}"
-        )["train"]
+        if use_registry:
+            data_files = (
+                registry_data_files
+                if registry_data_files
+                else f"{lang_oscar_id}/*.jsonl.gz"
+            )
+            self.ds = load_dataset(
+                "mhtoin/register_oscar",
```

Comment:
@ontocord would it be possible to document a bit more this dataset? currently there is no dataset card and we don't really know how this was generated, labeled, processed. Can we assume there were no modifications made to the original Oscar dataset?